README.md

```diff
@@ -10,7 +10,6 @@
 - 🌱 I'm currently learning [php](https://www.php.net)
 - ✉️ Best Medium to contact me is via [mail](mailto:arsenic.secondary@gmail.com)
 - 👯 I’m looking to collaborate with other open source enthusiasts on some 😎cool projects
-- 🥅 2022 Goals: increase my average typing speed to 100+ wpm ( currently at 77 wpm )
```

Comment:
Due to a lot of reasons, I didn't got enough time to work on my yearly goal. Would reset it once I get my sorted this year.